MACHINE-LEARNING-SERVICE/src/routes/objectRecognition.routes.js

```diff
@@ -0,0 +1,11 @@
+const {Router} = require('express');
+
+const {recognizeObjects, downloadImage} = require('../controllers/objectRecognition.controller');  
+const {uploadImagesMiddleware} = require('../middlewares/multer.middleware');
+const router = Router();
+
+// Routes uris
```

Comment:
omit the comment

Reply:
Please check again. I've already removed the comments.